core/indexer/process/transactions/utils.go

```diff
@@ -0,0 +1,88 @@
+package transactions
```

Comment:
What does this file should contain? Utils stuff. Please split this in files named `converters`, `checkers` and so on.

Reply:
removed this file.